Resources/doc/usage/autoinline.rst

```diff
@@ -0,0 +1,24 @@
+Disable auto inline
+===================
+
+By default, CKEditor enables the auto inline feature meaning that any
+``contenteditable`` attribute sets to ``true`` will be converted to CKEditor
+instance automatically. If you want to disable it, you can do it in your
+configuration or in your widget:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # app/config/config.yml
+        ivory_ck_editor:
+            auto_inline: false
+
+    .. code-block:: php
+
+        $builder->add('field', 'ckeditor', array('auto_inline' => false));
```

Comment:
as these 2 examples are showing different things (config vs widget), I recommend to use 2 seperate code blocks/examples. the tabbed code blocks (configuration block) is often used to show the same feature in different formats.